peps/pep-0777.rst

```diff
@@ -211,6 +238,14 @@ contents. Wheel revisions shouldn't cause package installations to break on
 older CPython revisions, as not only would it be frustrating, it would be
 incredibly hard to debug for users.
 
+This PEP relies on resolvers being able to efficiently acquire package
+metadata, usually through :pep:`658`. This might present a problem for users of
+package indices that do not serve :pep:`658` metadata. However, today most
```

Comment:
The text is not clear why the "However" sentence is relevant to the reliance on acquiring metadata via 658 or not.  Perhaps elaborate on the connection between range requests and efficient access to the metadata?